assets/src/edit-story/components/checklist/checks/storyPosterAttached.js

```diff
@@ -34,46 +42,32 @@ export function storyHasNoPosterAttached(story) {
 export function StoryPosterAttached() {
   //@TODO refine this context selector and storyHasNoPosterAttached to run more selectively
   const story = useStory(({ state }) => state);
-  return storyHasNoPosterAttached(story)
-    ? // <ChecklistCard
-      //   title={__('Add Web Story poster image', 'web-stories')}
-      //   titleProps={{
-      //     onClick: () => {
-      //       /* perform highlight here */
-      //     },
-      //   }}
-      //   footer={
-      //     <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
-      //       <ul>
-      //         <li>
-      //           {__('Use as a representation of the story.', 'web-stories')}
-      //         </li>
-      //         <li>{__('Avoid images with embedded text.', 'web-stories')}</li>
-      //         <li>
-      //           {__("Use an image that's at least 640x853px.", 'web-stories')}
-      //         </li>
-      //         <li>{__('Maintain a 3:4 aspect ratio.', 'web-stories')}</li>
-      //       </ul>
-      //       <Link
-      //         href={'#' /* figure out what this links to */}
-      //         size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
-      //       >
-      //         {'Learn more'}
-      //       </Link>
-      //     </Text>
-      //   }
-      //   cta={
-      //     <Button
-      //       size={BUTTON_SIZES.SMALL}
-      //       type={BUTTON_TYPES.SECONDARY}
-      //       onClick={() => {
-      //         /* perform highlight here */
-      //       }}
-      //     >
-      //       {__('Upload', 'web-stories')}
-      //     </Button>
-      //   }
-      // />
-      null
-    : null;
+  const setHighlights = useHighlights(({ setHighlights }) => setHighlights);
+
+  return storyHasNoPosterAttached(story) ? (
+    <ChecklistCard
+      title={PRIORITY_COPY.storyMissingPoster.title}
+      footer={
+        <FooterText>
+          <CardListWrapper>
+            <List size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
+              {PRIORITY_COPY.storyMissingPoster.footer}
+            </List>
+            <LearnMoreLink />
+          </CardListWrapper>
+        </FooterText>
+      }
+      cta={
+        <DefaultCtaButton
+          onClick={() =>
```

Comment:
🤔 currently, you can click the title for this functionality right? We should keep that, so also pass `titleProps={{ onClick: ... }}`

Reply:
Yea I wasn't sure what we wanted to do here. I guess clicking for highlight makes sense in single issue cards, but doesn't really make sense for multi issue cards.

when you have multiple thumbnails associated with an issue, I'm assuming we don't want clicking the title to do anything? I have the thumbnails highlighting associated elements on click in that scenario.

Reply:
but awesome. I'll add the same functionality to the title as the upload button here

Reply:
I feel like it should be consistent but I'm not sure the best path. 

multiple issues: no highlight click on title 
single issue: highlight click also on title OR if there's a thumbnail we just never do the title click? 

For now let's do the highlight click also on the title. Kinda keep the same functionality patterns where we can across the board and revisit as needed?